wpeverest/user-registration
# UR-622 Enhancement - Setup wizard design

### All Submissions:






### Changes proposed in this Pull Request:
Design changes according to the new design has been implemented, along with consent based tracking notice, some of the options refactor has been done.

### How to test the changes in this Pull Request:

1. Check if setup wizard design is as per the new design.
2. Check if the content based tracking notice popup and it is being saved properly.
3. Check if settings has been refactored as per our discussuion.
4. Check if install pages notice is displayed when setup wizard is skipped.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Setup wizard design.
